main/src/cgeo/geocaching/log/LogCacheActivity.java

```diff
@@ -741,11 +717,7 @@ private String getImageTitle(final Image image, final int position) {
         if (!StringUtils.isBlank(image.getTitle())) {
             return image.getTitle();
         }
-        String titlePrefix = logImageTitlePrefix.getText().toString();
-        if (StringUtils.isBlank(titlePrefix)) {
-            titlePrefix = getString(R.string.log_image_titleprefix);
-        }
-        return titlePrefix + " " + (position + 1);
+        return getString(R.string.log_image_titleprefix) + " " + (position + 1);
```

Comment:
Why does this return still contain `log_image_titleprefix`?

Reply:
> Why does this return still contain `log_image_titleprefix`?

Images for which you don't set a caption in the "image" dialog will get the log_title_prefix plus an index as their name (e.g. "Image 1" or "Image 3"). Ths behaviour is similar to the naming of waypoints extracted from PN as "Personal Note 1" and similar (except for images you can explicitely choose a different name in the image dialog if you want)

Reply:
Thanks, now I got it. :)